### ISSUE TITLE
Add custom list of tags to close

### ISSUE THREAD

**Issue by djtorrey** [formulahendry/vscode-auto-close-tag#69]
Opposite of the blacklist "do not auto close these tags", there should be an ability to define a list of tags you want to close, where the user adds in the settings what the opening tag and closing tags should be.

This would allow for detailed customization for instance, for templating languages that may not be supported out of the box.  For instance:

```
"auto-close-tag.customTagList":{
   "{{":"}}",
   "{%":"%}"
   "{% if %}":"{% endif %}"
}

```
The plugin would then check for the custom tag list, read the value, and auto complete with it.

**Comment by formulahendry** [comment]
This extension is targeting on HTML/XML tag.
Do you want to type `{{`, then `}}` will be auto inserted?

**Comment by djtorrey** [comment]
yes, ideally you would type {{ then }} would auto insert

**Comment by praenubilus** [comment]
Yes, I think if we can have this feature that will be really useful. In our editing(markdown/latex and other non-coding text), there may be lots of customized open/close parings. We can set this in workspace settings for each project.
